src/styles/card.scss

```diff
@@ -0,0 +1,75 @@
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 200px));
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  border: $gray-border;
+  padding: 32px 16px 16px;
+  border-radius: $rounding;
+  transition: $transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    margin-bottom: 40px;
+    padding-inline: 3px;
+    min-height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &:hover .card__title {
+    color: $title-hover;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: $secondary;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    font-size: 10px;
+    color: $main-accent;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    line-height: 18px;
+    color: $secondary;
+  }
+
+  &__price-number {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-accent;
```

Comment:
The variable `$main-accent` is used here and is correctly defined in `variables.scss` as `rgba(6, 11, 53, 1)`.